common/crypto/verify_ias_report/ias-certificates.h.in

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+extern const char ias_report_signing_ca_cert_pem[] =
+R"MLT(${INTEL_ROOT_CA_PEM}
+)MLT"
+;
+#ifdef __cplusplus
+}
+#endif
```

Comment:
i still dont understand why this is necessary. 
you can build the file in the cmake completely without using a template.